stores/typeorm-store/src/database.ts

```diff
@@ -33,18 +33,18 @@ export class TypeormDatabase {
         let con = await createConnection(cfg)
         try {
             let height = await con.transaction('SERIALIZABLE', async em => {
-                await em.query(`CREATE SCHEMA IF NOT EXISTS ${this.statusSchema}`)
+                await em.query(`CREATE SCHEMA IF NOT EXISTS "${this.statusSchema}"`)
                 await em.query(`
-                    CREATE TABLE IF NOT EXISTS ${this.statusSchema}.status (
+                    CREATE TABLE IF NOT EXISTS "${this.statusSchema}.status" (
```

Comment:
Here, it should be `"${this.statusSchema}".status`.

Also, I think we should restrict `statusSchema` to match `/^[a-z][a-z0-9_]*$/`.